.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Change description
+
+> Description here
+
+## Type of change
+
+- [ ] Bug fix (fixes an issue)
+- [ ] New feature (adds functionality)
+
+## Related issues
+
+Does this PR fix a Github Issue?
+
+> Fix [#1]()
+
+## Checklists
+
+### Development
+
+- [ ] Application changes have been tested thoroughly
```

Comment:
"thoroughly" is a weird word.  Our guidelines say that sometimes we don't need test too.  Perhaps "appropriately" is a better word?